WordPress/Classes/ViewRelated/Jetpack/Server Credentials/ServerCredentialsViewController.swift

```diff
@@ -0,0 +1,98 @@
+import CocoaLumberjack
+import Foundation
+import WordPressShared
+
+class ServerCredentialsViewController: UITableViewController {
+
+    // MARK: - Private Properties
+
+    private var blog: Blog!
+    private var service: JetpackScanService!
+    private lazy var handler: ImmuTableViewHandler = {
+        return ImmuTableViewHandler(takeOver: self)
+    }()
+
+    // MARK: - Initialization
+
+    convenience init(blog: Blog, service: JetpackScanService) {
+        self.init(style: .grouped)
+        self.blog = blog
+        self.service = service
+    }
+
+    // MARK: - View Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = NSLocalizedString("Remote Server Credentials", comment: "Title for the Jetpack Remote Server Credentials Screen")
+        ImmuTable.registerRows([EditableTextRow.self], tableView: tableView)
+        WPStyleGuide.configureColors(view: view, tableView: tableView)
+        reloadViewModel()
+    }
+
+    // MARK: - Model
+
+    private func reloadViewModel() {
+        // FIXME: Pass credentials retrieved from getCredentials
+        self.handler.viewModel = credentialsViewModel()
+
+        /*
+        service.getCredentials(for: blog, success: { [unowned self] credentials in
+            guard let credentials = credentials?.first else {
+                return
+            }
+            self.handler.viewModel = self.credentialsViewModel(credentials)
+            DDLogInfo("Fetched Remote Server Credentials")
+        }, failure: { error in
+            DDLogError("Error while fetching Remote Server Credentials: \(error.localizedDescription)")
+        })
+        */
+    }
+
+    private func credentialsViewModel(_ credentials: JetpackScanCredentials? = nil) -> ImmuTable {
```

Comment:
Passing an optional `JetpackScanCredentials` for now, until I'm able to hook up the endpoints